library/alloc/src/vec/spec_from_iter_nested.rs

```diff
@@ -1,6 +1,8 @@
 use core::iter::TrustedLen;
 use core::ptr::{self};
 
+#[allow(unused_imports)]
+use super::SpecFromIter;
```

Comment:
The more expected fix is to change this line
https://github.com/rust-lang/rust/blob/dcf622eb70aebe16d40c5f88fa2a41fa7019541c/library/alloc/src/vec/spec_from_iter_nested.rs#L8
to 
```
/// see [`SpecFromIter`](super::SpecFromIter) for details. 
```

Reply:
My bad. Thanks for the help @lzutao